Formula/wxpython.rb

```diff
@@ -60,11 +60,15 @@ def install
       # OpenGL and stuff
       "BUILD_GLCANVAS=1",
       "BUILD_GIZMOS=1",
-      "BUILD_STC=1"
+      "BUILD_STC=1",
     ]
 
     cd "wxPython" do
       system "python", "setup.py", "install", "--prefix=#{prefix}", *args
     end
   end
+
+  test do
+    system "python -c 'import wx ; print wx.version()'"
```

Comment:
We prefer to wrap to 80 characters as much as possible. Haven't checked but I'd be surprised if that wasn't over. There are various ways to wrap to the 80, but the block I proposed is the most common one in use, at least till we change our minds 😄.